evaluator/autoware_planning_evaluator/include/autoware/planning_evaluator/metric_accumulators/planning_factor_accumulator.hpp

```diff
@@ -108,14 +108,18 @@ class PlanningFactorAccumulator
       distance_to_stop(0.0),
       stop_decision_keep_time_accumulator(),
       state_updated(false),
-      state_Initialized(false)
+      state_initialized(false)
     {
     }
 
     void disable_current_stop_decision()
```

Comment:
```suggestion
    void clear_current_stop_decision()
```
from the implemantation, above function name seems reasonable.